apps/website/components/button.tsx

```diff
@@ -0,0 +1,39 @@
+export interface ButtonProps {
+  children: string;
+  className?: string;
+  icon?: React.ElementType;
+  type: "primary" | "secondary";
+}
+
+const Button = ({
+  children,
+  className = "",
+  icon: Icon,
+  type,
+}: ButtonProps): JSX.Element => {
+  let typeClasses = "";
+  if (type === "primary") {
+    typeClasses = "text-white hover:bg-gruen bg-blau";
+  } else if (type === "secondary") {
+    typeClasses =
+      "text-blau border border-blau hover:text-gruen hover:border-gruen bg-white";
+  }
+  return (
+    <button
+      className={`px-5 rounded-3xl font-text min-h-12 flex items-center ${typeClasses} ${className}`}
+      data-testid="button"
+    >
+      {Icon && (
+        <Icon
+          height={24}
```

Comment:
please add a facade

`height={3 * baseHeight}`
Therefore, please introduce a file named constants.ts in the libs folder and end export the `baseHeight` with the value of 8.